control-plane/csi-driver/src/bin/node/nodeplugin_grpc.rs

```diff
@@ -76,6 +76,7 @@ impl NodePluginGrpcServer {
             endpoint
         );
         Server::builder()
+            .timeout(std::time::Duration::from_secs(30))
```

Comment:
Be mindful that the gRPC themselves have timeouts, setting this value needs to be done carefully.
I suggest not doing this until we figure out the correct value.

```suggestion
```

Reply:
Done.

Reply:
> You mean the client side timeout?

Yes